src/js/statistics/dgva-statistics.js

```diff
@@ -65,73 +72,37 @@ DgvaStatistics.prototype = {
-        var dataArray = data.chartData;
-        var title = data.title;
-        $(function () {
-            Highcharts.setOptions({
-                colors: ['#207A7A', '#2BA32B', '#2E4988', '#54BDBD', '#5DD15D', '#6380C4', '#70BDBD', '#7CD17C', '#7D92C4', '#295C5C', '#377A37', '#344366', '#0A4F4F', '#0E6A0E', '#0F2559' ],
-                chart: {
-                    style: {
-                        fontFamily: 'sans-serif;'
-                    }
-                }
-            });
-            $(render_id).highcharts({
-                chart: {
-                    plotBackgroundColor: null,
-                    plotBorderWidth: null,
-                    plotShadow: false,
-                    marginBottom:115
-                },
-                legend: {
-                    enabled: true,
-                    width: 100,
-                    // margin: 0,
-                    labelFormatter: function () {
-                        return '<div>' + this.name + '(' + this.y + ')</div>';
-                    },
-                    layout: 'vertical',
-                    useHTML: true,
-                    align: "center"
-                },
-                title: {
-                    text: 'Top 5 Studies <br>\u00A0<span style="font-size:12px;">by ' + title + '</span>',
-                    style: {},
-                    align: 'center'
-                },
-                tooltip: {
-                    pointFormat: '<b>{point.y}</b>'
-                },
-                plotOptions: {
-                    pie: {
-                        allowPointSelect: true,
-                        cursor: 'pointer',
-                        dataLabels: {
-                            enabled: false
-                        },
-                        showInLegend: true
-                    }
-                },
-                series: [
-                    {
-                        type: 'pie',
-                        name: 'Top 5 Studies by ' + title,
-                        data: dataArray
-                    }
-                ],
-                credits: {
-                    enabled: false
-                }
-            });
+        var chartData = data.chartData;
+        chartData.unshift([label, 'count']);
+        var title = 'Top 5 studies by '+data.title;
+        // google.charts.load('current', {'packages':['bar']});
+        google.charts.load('current', {'packages':['corechart']});
+        google.charts.setOnLoadCallback(function(){
+            var data = google.visualization.arrayToDataTable(chartData);
+            var container = $(id),
+                width = container.width();
+            var options = {
+                title: title,
+                chartArea: {width: width, height:300,top:50},
+                // colors: ['#207A7A', '#2BA32B', '#2E4988', '#54BDBD', '#5DD15D', '#6380C4', '#70BDBD', '#7CD17C', '#7D92C4', '#295C5C', '#377A37', '#344366', '#0A4F4F', '#0E6A0E', '#0F2559' ],
```

Comment:
It seems possible to [configure the colors in a pie chart](https://developers.google.com/chart/interactive/docs/gallery/piechart) (search for "slices"). Could these be changed to match those in production?